Sources/StructuredQueriesMacros/TableMacro.swift

```diff
@@ -207,6 +208,24 @@ extension TableMacro: ExtensionMacro {
               queryValueType: columnQueryValueType
             )
 
+          case .some(let label) where label.text == "generated":
+            guard
+              let memberName = argument.expression.as(MemberAccessExprSyntax.self)?.declName
+                .baseName.text,
+              ["stored", "virtual"].contains(memberName)
+            else {
+              diagnostics.append(
+                Diagnostic(
+                  node: argument.expression,
+                  message: MacroExpansionErrorMessage(
+                    "Argument 'generated' must be '.stored' or '.virtual'"
+                  )
+                )
+              )
+              continue
+            }
```

Comment:
I think the type-checking done at the macro level is sufficient enough, so these diagnostics shouldn't be necessary.
```suggestion
```